### ISSUE TITLE
Transform control icons

### ISSUE THREAD

**Issue by SylvainCorlay** [jupytercad/JupyterCAD#585]
We should probably replace the placeholder for the transfor controls toggle icon before releasing.

![Image](https://github.com/user-attachments/assets/f342a7e6-a5ab-496c-a4df-08bd4c19e880)

**Comment by arjxn-py** [comment]
Yes, once we have a decision on new Icon we shall be able to do it.

**Comment by SylvainCorlay** [comment]
We could decide for a simple cross icon at first, that would be good enough for this version.

![Image](https://github.com/user-attachments/assets/c872f5e0-8aea-433e-a8aa-827b82430dc4)

**Comment by arjxn-py** [comment]
> We could decide for a simple cross icon at first, that would be good enough for this version.

Sounds good, i'm happy with this icon. Thanks :)

**Comment by SylvainCorlay** [comment]
We should probably have our own drawing that we own for this repository.
